docs/implementation.rst

```diff
@@ -59,3 +59,6 @@ Cheesecake project  ranks Python packages based on various empirical “kwalitee
 The final score depends on how well the package scores for all indexes listed above. The relative score  (percent of 
 points obtained compared to maximum possible points) can help the curator in determining  the goodness of the add-on.
 More details about the cheesecake package at (https://pypi.python.org/pypi/Cheesecake)
+
+References:  A sample package like (https://pypi.python.org/pypi/wincertstore/json) was referred for the field names,
```

Comment:
For link styles, you might want to follow this in the Plone docs styleguide: https://docs.plone.org/about/rst-styleguide.html#links